src/components/NanoContract/NanoContractDetails.js

```diff
@@ -133,7 +136,7 @@ export const NanoContractDetails = ({ nc }) => {
             )}
             keyExtractor={(item) => item.txId}
             refreshing={isLoading}
-            onRefresh={handleMoreTransactions}
+            onRefresh={handleNewTransactions} // pull gesture
```

Comment:
Ok, so this is the pull to refresh, but where is the code for the infinite scroll?

Reply:
Sorry, I forgot to upload. Here it is: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/512/commits/2040049f05e0125695c2ad5e612d54039da8b087